CHANGELOG.md

```diff
@@ -5,9 +5,22 @@ This project adheres to [Semantic Versioning 2](http://semver.org/).
 
 ## [Unreleased]
 
+## 1.1.0 - 2018-02-21
```

Comment:
Shouldn't this be a `2.0.0` release?

Reply:
Reaction to my previous comment:
`2.0.0` release is not required since there are now BC breaks in this PR. When using PHP 5.* you will still get version `1.0`.